src/recipe.js

```diff
@@ -6,11 +6,16 @@ class Recipe {
     this.tags = recipe.tags;
     this.ingredients = recipe.ingredients;
   }
-  calculateIngredientsCost() {
-    // return this.ingredients.map(i => {
-    //   ingredientData.find(ingredient => ingredient === i);
-    // });
-  }
+
+
+  calculateIngredientsCost(ingredients) {
+    const totalCost = this.ingredients.reduce((acc, currentIngredient) => {
+      let foundIngredient = ingredients.find(ingredient => ingredient.id === currentIngredient.id);
+      acc += currentIngredient.quantity.amount * foundIngredient.estimatedCostInCents;
+      return acc;
+    }, 0);
+    return totalCost / 100;
+  };
```

Comment:
nice work building this method out. Can't wait to add it in to check functionality!